scipy/fft/tests/test_real_transforms.py

```diff
@@ -199,6 +199,10 @@ def test_orthogonalize_noop(func, type, norm, xp):
 def test_orthogonalize_dct1(norm, xp):
     x = xp.asarray(np.random.rand(100))
 
+    # use array-api-compat namespace for dask
+    # since dask asarray never makes a copy
+    # which makes xp_copy silently a no-op
+    xp = array_namespace(x)
```

Comment:
could we adjust `xp_copy` to special-case and copy for Dask instead?

Reply:
We are currently passing in xp into ``xp_copy`` in these tests which is the problem.

i.e. ``xp_copy(arr, xp=<bad dask.array xp>)``

If we don't pass in xp_copy, I've changed ``array_namespace`` in our vendored array-api-compat to use the wrapped dask.array namespace by default (will upstream this later).

Reply:
It would be easier to just do these copies NumPy side so we don't need to involve `xp_copy` at all:

```python
x = np.random.rand(100)
x2 = x.copy()
x = xp.asarray(x)
x2 = xp.asarray(x2)
```

Reply:
We can also drop the xp argument to ``xp_copy`` (which would be a smaller diff).

Reply:
happy with that!

Reply:
actually, no - `x2` should be an array from the unwrapped namespace before it is passed into `func`.

Reply:
I think I've changed all of these back to using numpy, let me know if there's any more left.